packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -41,6 +41,11 @@ export const Docs: () => JSX.Element = () => {
     }
   }, [refTeamsMeeting]);
 
+  const teamsInteropFileSharingBulletinPoints =
```

Comment:
Is this being used anywhere?

Reply:
sorry it should've been removed, just removed it